app/uk/gov/hmrc/pensionschemereturn/models/requests/etmp/AccountingPeriodDetailsRequest.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pensionschemereturn.models.requests.etmp
+
+import play.api.libs.json.{Json, OWrites}
+
+import java.time.LocalDate
+
+case class AccountingPeriodRequest(
```

Comment:
For me it made sense to have these class names prefixed with etmp (as it was previously), as it was easier to understand if you were looking at some code standalone. But that is just a personal preference I think, so unless other people think the same, I don't think it is a strong enough reason to refactor them. Good shout to have them in a separate package though.